crates/test_utils/wasm/wasm_workspace/genesis_self_check_requires_properties/src/integrity.rs

```diff
@@ -0,0 +1,13 @@
+use hdi::prelude::*;
+
+#[hdk_extern]
+fn genesis_self_check(_: GenesisSelfCheckData) -> ExternResult<ValidateCallbackResult> {
+    let props = dna_info()?.modifiers.properties;
```

Comment:
Off-topic really, but can we not use `YamlProperties` in the HDI? That feels like it's something that should be available. If I can access the properties I should be able to easily deserialize them without pulling in a serialization library that might not be compatible

Reply:
You're proposing to use `YamlProperties` here, right? Not the opposite. Yes, I would endorse that instead of the serialized bytes.

Reply:
Seems reasonable

Reply:
Yes exactly that @jost-s, it's only available in the HDK and I think it's one of the things that should move